.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,26 @@
+name: CIFuzz
+on: [pull_request]
```

Comment:
To prevent this workflow running twice every-time the code is updated (following main.yml), I would prefer this to be
````
on:
  push:
    branches:
      - main
      - develop
      - release-*
      - gh-workflows
  pull_request:
    branches:
      - main
      - develop
````